vendor.conf

```diff
@@ -101,13 +101,13 @@ github.com/googleapis/gax-go da06d194a00e19ce00d9011a13931c3f6f6887c7
 google.golang.org/genproto d80a6e20e776b0b17a324d0ba1ab50a39c8e8944
 
 # containerd
-github.com/containerd/containerd 3addd840653146c90a254301d6c3a663c7fd6429
+github.com/containerd/containerd fc10004571bb9b26695ccbf2dd4a83213f60b93e https://github.com/dmcgowan/containerd.git
```

Comment:
We should wait for https://github.com/containerd/containerd/pull/1252 to get merged before merging this one 👼